### PR TITLE
WIP: Add yarn in PHP image

### PR DESCRIPTION
**Pull request by maxhelias**

### CODE REVIEW

File: php/Dockerfile [hunk 1]
```diff
@@ -70,8 +70,8 @@ RUN composer global require hirak/prestissimo --prefer-dist --no-progress --no-s
 RUN curl -sL https://deb.nodesource.com/setup_10.x | bash - && \
   apt-get install -y nodejs
 
-# Install Grunt CLI
-RUN npm install -g grunt-cli
+# Install Grunt CLI & Yarn
+RUN npm install -g grunt-cli yarn
```
Comment (ajardin) [inline php/Dockerfile]: Why using npm to install Yarn? x)

Comment (ker0x) [reply php/Dockerfile]: @maxhelias juste add it to the `apt-get install` list!

Comment (matthieupetel) [reply php/Dockerfile]: Why is yarn needed ? npm isn't enough ?

**Review by ker0x — APPROVED** [review]

File: php/Dockerfile (hunk 1 above)
Comment (ajardin) [reply php/Dockerfile]: @matthieupetel, because it seems to be faster than npm. Here is a recent [benchmark](https://github.com/appleboy/npm-vs-yarn).

**Review by matthieupetel — APPROVED** [review]

**Pull request closed without merging**